tests/manual-test-examples/p5vr/basic/example.js

```diff
@@ -28,9 +28,10 @@ function draw() {
 }
 
 function checkSetup() {
-  if(setupCounter !== 2) {
-    console.error('setup() running incorrectly');
-  }
+  // if(setupCounter !== 2) {
```

Comment:
I say remove this altogether. This can be introduced as a unit test rather than being included in the manual test.

Reply:
Removed with this commit: 02aba6760472cb0ea9cd759f53a446f3c691026c